api/v1/swagger.yaml

```diff
@@ -6941,6 +6941,148 @@ paths:
           $ref: "#/responses/genericError"
 
 # =============================================================================
+  /events:
+    x-swagger-router-controller: events
+    get:
+      summary: Retrieve all events
+      tags: [ events ]
```

Comment:
pls. add "description" attribute (a more verbose version of the "summary" attribute) to all of these new endpoints